articles/managed-instance-apache-cassandra/monitor-clusters.md

```diff
@@ -166,6 +169,29 @@ Examples:
   cassandra@cqlsh> LIST ROLES;
   ```
 
+### Cassandra 4 and later
+In Cassandra 4 and later, you can configure your whitelist in the Cassandra configuration. For detailed guidance on updating the Cassandra configuration, please refer to [Update Cassandra Configuration](https://learn.microsoft.com/en-us/azure/managed-instance-apache-cassandra/create-cluster-portal#update-cassandra-configuration). The available options are as follows:
```

Comment:
Potentially reference upstream audit logging docs: https://cassandra.apache.org/doc/stable/cassandra/operating/audit_logging.html

Reply:
Updated, could you please review it again ?